pkg/kubelet/kubelet.go

```diff
@@ -1833,6 +1833,12 @@ func (kl *Kubelet) syncLoop(updates <-chan kubetypes.PodUpdate, handler SyncHand
 		factor = 2
 	)
 	duration := base
+	// Responsible for checking limits in resolv.conf
```

Comment:
Worth making it clear in the comment that since we are calling this in `syncLoop`, we don't need to call it anywhere else?